templates/create-daml-app/ui/public/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<!-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-daml-app"
+    />
+    <link rel="apple-touch-icon" href="logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
Do we really want to have this in the GTG? Seems like useless noise to me.

Reply:
It is extremely noisy. It looks like this whole directory only exists to construct the header with the DAML logo?

Reply:
This is all coming from `create-react-app`. We can strip a lot of this.